README.md

```diff
@@ -0,0 +1,36 @@
+# HW14 Travel Agency Website
+
+## Project Overview
+A modern, three-page website for a travel agency dedicated to showcasing one country. The site is built using SCSS, HTML, and Bootstrap with advanced features like parallax effects, tabs, a lightbox slider, and CSS animations. The site is responsive and includes a fixed navigation bar, interactive map, and a contact form.
+
+## Technologies Used
+- **HTML5**: For structuring the website content.
+- **SCSS**: Modular stylesheets with variables, mixins, and imports. Compiles into one common CSS file for all pages.
+  - `tools.scss`: Contains variables and mixins.
+  - `reset.scss`: For CSS reset.
+- **Bootstrap**: For grid layout, responsiveness, and basic components.
+- **Font Awesome**: Icons used for features and social media links.
+- **JavaScript**: Used for implementing tabs, lightbox slider, and parallax scrolling effects.
+- **Google Maps API**: Integrated for displaying interactive maps on the country page.
+- **GitHub**: Version control with at least 10 commits.
+- **Free Hosting**: The site is deployed on a free hosting service for public access.
+
+## Key Features
+1. **Parallax Effects**: Smooth scrolling background images on key sections.
+2. **Lightbox Slider**: Fullscreen image viewer with smooth transitions.
+3. **Tabs**: JavaScript-driven tabs for displaying organized content.
+4. **Fixed Navigation**: The header menu remains visible while scrolling.
+5. **Interactive Map**: Google Maps integration to display key locations.
+6. **CSS Animations**: Hover effects and transitions on buttons and cards.
+7. **Responsive Design**: Site adapts to different screen sizes using percentages for widths and Bootstrap's grid system.
+8. **Contact Form**: Includes form validation with smooth submission feedback.
+
+## Project Structure
+- **Home Page**: Overview of the destination with popular places, customer reviews, and a call to action.
+- **Country Page**: Detailed information about the country, recommended itineraries, and an interactive map.
```

Comment:
+ Tour description page

Reply:
`## Project Structure
- **Home Page**: Overview of the destination with popular places, customer reviews, and a call to action.
- **Tour Description Page**: Detailed information about the country, recommended itineraries, and an interactive map.`